Utilities.py

```diff
@@ -66,3 +68,15 @@ def GetFieldFromEmbed(embed, field_name):
     for field in embed_dict["fields"]:
         if field["name"].lower() == field_name.lower():
             return field["value"]
+
+def CalculatePercentage(total, actual):
+    return "{:.2f}".format((actual * 100) / total)
+
+def GetMembersFromMention(mention_string):
+    members_mentions_list = [mention + ">" for mention in mention_string.split(">") if mention]
+    members_list = []
+    for member_mention in members_mentions_list:
+        member = member_db.GetMemberByID(int(re.match(r'<@!?(\d+)>', member_mention).group(1)))
+        if member != None:
+            members_list.append(member)
+    return members_list
```

Comment:
This can be simplified greatly by re.findall

Reply:
Done